Horizon/Horizon/Sources/Features/Dashboard/View/DashboardView.swift

```diff
@@ -127,5 +127,5 @@ struct DashboardView: View {
 }
 
 #Preview {
-    DashboardView(viewModel: .init(interactor: GetProgramsInteractor()))
+    DashboardView(viewModel: .init(interactor: GetProgramsInteractorLive(appEnvironment: .shared)))
```

Comment:
Can you create a mock interactor so we have working preview with fake data? Check `AssignmentDueDatesAssembly` for guidance.